base/sparse/cholmod.jl

```diff
@@ -790,9 +805,19 @@ function sparse{Ti}(A::Sparse{Complex{Float64},Ti}) # Notice! Cannot be type sta
     end
     return convert(Hermitian{Complex{Float64},SparseMatrixCSC{Complex{Float64},Ti}}, A)
 end
-sparse(L::Factor)   = sparse(Sparse(L))
+function sparse(L::Factor)
+    A = Sparse(L)
+    A*A'
```

Comment:
Ah, thank you, I missed all of those subtleties. If I understand correctly, you're fine with changing it as long as I get the logic right?